.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - run: |
+        WATTSI_VERSION=$(git rev-list --count HEAD)
+        echo "$WATTSI_VERSION" > src/version.inc
+        echo "::set-env name=WATTSI_VERSION::$WATTSI_VERSION"
+    - uses: docker/build-push-action@v1
+      with:
+        registry: docker.pkg.github.com
```

Comment:
I'd like to continue using Docker Hub and not leave https://hub.docker.com/repository/docker/whatwg/wattsi/ orphaned, if possible...

Reply:
It is possible! Can you add `DOCKER_USERNAME` and `DOCKER_PASSWORD` secrets accessible to (at least) this repo and html-build? Then I can wire things up.

Reply:
Done! Although I named them `DOCKER_HUB_USERNAME` and `DOCKER_HUB_PASSWORD` for slightly better accuracy.

I also created a new account instead of reusing mine. I hope I got all the permissions set up right.